site/styles/_plans.scss

```diff
@@ -92,7 +92,8 @@ s72-plan-label {
   white-space: nowrap;
   width: 1px;
 
-  @include media-breakpoint-up(sm) {
+  @include media-breakpoint-up(md) {
     display: block;
+    margin: 4px 0 0 18px;
```

Comment:
That 18px padding causes the label to be offset 
![image](https://user-images.githubusercontent.com/32156362/170914727-ed72b01c-3c7f-4ce8-96e4-3e810c22e7bf.png)

Reply:
That's how it is in [the design](https://app.zeplin.io/project/607f961cf9f03790cc66e475/screen/6205cb31c677b1be9ce9955c).  Is that wrong?

Reply:
If I was impersonating a designer, I'd suggest it's probably meant to be centered, but there isn't a good solution for centering with translations. I thought there was some discussion about a left-margin and the text flowing under the other buttons?

Reply:
I didn't consider that they might just want it centered.  RE text flowing under the buttons... it works fine for long translations with this implementation!

Reply:
Yeah that's what I meant 👍

Reply:
> If I was impersonating a designer, I'd suggest it's probably meant to be centered, but there isn't a good solution for centering with translations. I thought there was some discussion about a left-margin and the text flowing under the other buttons?

Theo and I figured out a strategy to make it centered until its the width of the subscribe button. Then it grows to the right.